mozilla-release/browser/base/content/tabbrowser.js

```diff
@@ -5519,18 +5519,24 @@ var TabContextMenu = {
     const windowIsPrivate = PrivateBrowsingUtils.isWindowPrivate(window);
     const tabIsPrivate = this.contextTab.private;
     const togglePrivateItem = document.getElementById("context_togglePrivate");
-    togglePrivateItem.hidden = windowIsPrivate;
-    togglePrivateItem.label =
-        gNavigatorBundle.getString(
-            tabIsPrivate ? "apt.tabContext.reloadInNormalMode"
-                         : "apt.tabContext.reloadInForgetMode");
     const addExceptionItem =
         document.getElementById("context_togglePrivateAndRememberDomain");
-    addExceptionItem.hidden = windowIsPrivate || !autoForgetTabs.isActive();
-    addExceptionItem.label =
-        gNavigatorBundle.getString(
-            tabIsPrivate ? "apt.tabContext.alwaysInNormalMode"
-                         : "apt.tabContext.alwaysInForgetMode");
+    if (tabIsPrivate === null) {
+      togglePrivateItem.hidden = true;
+      addExceptionItem.hidden = true;
+    }
```

Comment:
@alver-cliqz could you please put "else" right after "}" at the same line?
So code style would be more consistent.

Reply:
good point, done